source/d_threedi_model.rst

```diff
@@ -5,158 +5,158 @@ Running a 3Di model
+The functioning of these buttons is described in the following sections.
 
-.. figure:: image/d3.1_point_vs_cross_section.png
-	:alt: Point (left) or cross section (right) selection
+NOTE: The result of forcing water is not visible until the simulation is running.
 
-The buttons at the bottom left of the screen are used to interactively adjust the forcing of the model. The functioning of these buttons is described in the following sections.
+	
+Discharge or pump point
+----------------------------
 
-NOTE: The result of forcing water is not visible until the simulation is running. The color scheme of the water depth is adjustible through the :ref:`layers_menu` under *Animation* at *WMS color range*.
+With the discharge icon a constant source (or sink in case of a pump) of water can be added to the model. Select the icon and click at a location on the map to point the location. You can then change the rate (in m3/s) you want to apply. The water that is taken out of the model will not flow back into the model and is considered a loss. 
 
 Rainfall
 --------
 
-Through the precipitation icon rainfall can be added to the model. When clicking on the map, a blue circle and the icon of a raincloud appear on the map. The size of the raincloud is proportional to the size of the window. Zooming out will result in a larger raincloud and vice versa. Using the menu at the bottom left of the screen the intensity of rainfall can be adjusted.
-
-The above example is about the so-called *local only* rain event. By pressing the *additional settings* button (wrench icon) a new menu opens with different options for the rainfall:
+Through the precipitation icon rainfall can be added to the model. The following options are available:
 
-* **Radar**: use historical rainfall data.
+* **Radar**: use historical rainfall data (only available in the Netherlands).
 * **Design**: use a design event. This event is homogeneous over the entire model area and heterogeneous in time.
 * **Constant**: a homogeneous event in both space and time across the entire model range.
 
-These three options for adding rainfall all cover the entire model area, in contrast to the *local only* event.
+These three options for adding rainfall all cover the entire model area
 
-When the rainfall is active a cloud icon appears on the bottom right of the screen. Information about the rainfall event can be accessed by clicking this icon.
+When the rainfall is active a cloud icon appears on the top right of the screen. Information about the rainfall event can be accessed by keeping the rainfall tab open. Active and past (inactive) events are shown in this tab 
 
 .. figure:: image/d3.2_rainfall.png
 	:alt: Rainfall event
+	
+The numbers of the design rainfall point the following rainfall events:
+Rain 3 till 10 are the Dutch C2100 rainfall events
+Rain 11 till 16 are the DPRA rainfall events
 
```

Comment:
Remove empty lines after all the text. There are cases where this either renders weird, or even throws errors during tests.